<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B7K6OT8" failures="1" tests="5" name="test1" time="0.082" errors="0" timestamp="05 May 2019 15:54:50 GMT">
  <testcase classname="sele1.sample.NewTest" name="f" time="0.012"/>
  <testcase classname="sele1.sample.NewTest1" name="f" time="0.004"/>
  <testcase classname="sele1.sample.NewTest2" name="f" time="0.002"/>
  <testcase classname="sele1.sample.NewTest3" name="f" time="0.002"/>
  <testcase classname="sele1.sample.NewTest4" name="f" time="0.003">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at sele1.sample.NewTest4.f(NewTest4.java:10)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- test1 -->
